packages/mds-types/transformers.ts

```diff
@@ -0,0 +1,87 @@
+import { UUID, Timestamp, VEHICLE_EVENT, Telemetry, VEHICLE_REASON } from './index'
+
+export interface VehicleEvent_v0_4_1 {
+  device_id: UUID
+  provider_id: UUID
+  timestamp: Timestamp
+  timestamp_long?: string | null
+  delta?: Timestamp | null
+  event_type: VEHICLE_EVENT
+  event_type_reason?: VEHICLE_REASON | null
+  telemetry_timestamp?: Timestamp | null
+  telemetry?: Telemetry | null
+  trip_id?: UUID | null
+  service_area_id?: UUID | null
+  recorded: Timestamp
+}
+
+export interface VehicleEvent_v1_0_0 {
+  device_id: UUID
+  provider_id: UUID
+  timestamp: Timestamp
+  timestamp_long?: string | null
+  delta?: Timestamp | null
+  vehicle_state: VEHICLE_EVENT
+  event_types: VEHICLE_REASON[]
+  telemetry_timestamp?: Timestamp | null
+  telemetry?: Telemetry | null
+  trip_id?: UUID | null
+  service_area_id?: UUID | null
+  recorded: Timestamp
+}
+
+export function v0_4_1_to_v1_0_0(event: VehicleEvent_v0_4_1): VehicleEvent_v1_0_0 {
```

Comment:
I think we ought to name these such that they dictate what data they're manipulating (e.g. `VehicleEvent_v0_4_1_to_v1_0_0`), would also make sense to have a transformers folder as opposed to just a file, I think

Reply:
Sounds good. What do you think of having different versions for each type? E.g. `VehicleEvent_v4_0_0`. It feels horrible but I don't see a way around it either.

Reply:
I think we should probably have the different versions for each type for the older ones (e.g. `VehicleEvent_v0_4_0`, `VehicleEvent_v0_4_1`, `VehicleEvent` (latest, 1.0.0))

Then whenever we do a version bump we can mark the old ones with their version number (and if there aren't changes, just have them be an alias or something)